app/templates/app/projects.html

```diff
@@ -3,105 +3,118 @@
-      <div class="project-border">
-      <div class="project-header">
-        <div class="project-row">
-            <div class="col-sm-6 project-left-col">
-              <h3>
-                <a class="project-name" href="{% url 'project_detail' project.project.id %}">
-                    {{ project.project.name }}
-                </a>
-              </h3>
-            <div class="card-text-project">
-                <a href="{{ project.project.url }}" target="_blank" class="project-url">{{ project.project.url }}</a>
-            </div>
-            <div class="card-text-project project-text">
-                {{ project.institution_name }}
-            </div>
-          </div>
-          <div class="col-sm-6 project-right-col">
-            {% if project.logo %}
-            <img src="{{ project.logo.url }}" alt="{{ project.name }} logo"
-                 class="project-logo">
-            {% endif %}
-          </div>
-        </div>
-      </div>
-      <div class="project-body">
-        <div class="card-text-description">
-            {{ project.description|truncatewords:30 }} <a href="{% url 'project_detail' project.project.id %}">Read more</a>
-        </div>
+<div class="project-list">
+  {% for project in projects %}
+  <div class="project-border">
+  <div class="project-header">
+    <div class="project-row">
+        <div class="project-left-col">
+          <h3>
+            <a class="project-name" href="{% url 'project_detail' project.project.id %}">
+                {{ project.project.name }}
+            </a>
+          </h3>
         <div class="card-text-project">
-          <span class="icon-text"><i class="project-icon bi-vector-pen"></i></span>
-          <span class="icon-text">{{ project.languages }}</span>
+            <a href="{{ project.project.url }}" target="_blank" class="project-url" title="External link">
+                {{ project.project.url }}
+            </a>
         </div>
-        <div class="card-text-project">
-            <span class="icon-text"><i class="project-icon bi-book"></i></span>
-            <span class="icon-text">{{ project.subjects }}</span>
+        <div class="card-text-project project-text">
+            {{ project.institution_name }}
         </div>
-        <div class="card-text-project">
-          <span class="icon-text"><i class="project-icon bi-calendar3"></i></span>
-          <span class="icon-text">{{ project.date }}</span>
-        </div>
-
       </div>
+      <div class="project-right-col">
+        {% if project.logo %}
+        <img src="{{ project.logo.url }}" alt="{{ project.name }} logo"
+             class="project-logo">
+        {% endif %}
       </div>
-
-      <br />
-      <br />
-      <br />
-      {% endfor %}
     </div>
+  </div>
+  <div class="project-body">
+    <div class="card-text-description">
+        {{ project.description|truncatewords:30 }} <a href="{% url 'project_detail' project.project.id %}">Read more</a>
```

Comment:
Please mark "Read more" for translation.

Reply:
will do